acceptance-test/build.gradle

```diff
@@ -36,7 +36,7 @@ configurations {
 dependencies {
     ext {
         dropwizardVersion = '3.1.3'
-        curatorVersion = '4.2.0'
+        curatorVersion = '5.1.0'
```

Comment:
It makes sense to use the same variable in root `build.gradle`, like `kafkaClientVersion`

Reply:
👍